mkdocs.yml

```diff
@@ -8,6 +8,30 @@ theme:
   language: 'en'
   features:
     # - navigation.top
+
+  # Respect the users default settings and add adjustable light/darkmode toggle.
+  # taken from https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#system-preference
+  palette:
+
+    # TODO: enable this after we upgrade mkdocs-material above "^9.5"
+    # Palette toggle for automatic mode
+    # - media: "(prefers-color-scheme)"
+    #   toggle:
+    #     icon: material/brightness-auto
+    #     name: Switch to light mode
+    # Palette toggle for light mode
```

Comment:
(Very minor): Instead of having three comments like this, you could just change line 12 to say something like "...and add a toggle for manually choosing automatic/light/dark palette."

Reply:
done